ray-operator/controllers/ray/common/pod.go

```diff
@@ -244,11 +267,11 @@ func convertCmdToString(cmdArr []string) (cmd string) {
 	return cmdAggr.String()
 }
 
-func getRayContainerIndex(pod v1.Pod) (rayContainerIndex int) {
+func getRayContainerIndex(podSpec v1.PodSpec) (rayContainerIndex int) {
```

Comment:
Changed the signature so that you can use pod or pod template by passing in its .Spec member.